mobx/lib/src/core/computed.dart

```diff
@@ -146,21 +146,28 @@ class Computed<T> extends Atom implements Derivation, ObservableValue<T> {
 
   bool _isEqual(T x, T y) => x == y;
 
+  // [fireImmediately] never did quite what it sounded like, and the handler has always
+  // fired immediately. So it is deprecated and now non-functional.
   Function observe(void Function(ChangeNotification<T>) handler,
-      {bool fireImmediately}) {
+      {@deprecated bool fireImmediately}) {
     var firstTime = true;
     T prevValue;
 
     return autorun((_) {
       final newValue = value;
-      if (firstTime == true || fireImmediately == true) {
-        _context.untracked(() {
-          handler(ChangeNotification(
-              type: OperationType.update,
-              object: this,
-              oldValue: prevValue,
-              newValue: newValue));
-        });
+
+      void handleChange() {
+        handler(ChangeNotification(
+            type: OperationType.update,
+            object: this,
+            oldValue: prevValue,
+            newValue: newValue));
+      }
+
+      if (firstTime == true) {
+        _context.untracked(handleChange);
+      } else {
+        handleChange();
```

Comment:
So I'm not clear on what work `untracked` is doing here. For some reason it felt like having the handler be untracked in the initial setup was desired, but I'm really not sure what was supposed to happen in the alternate case. This was my best guess as to the intention, but that is based on essentially nothing. So I'm certainly interested in guidance here.

Reply:
Since we are using `autorun`, `untracked` ensures we are not going into a self-loop because of observing  other observables invoked as part of the `handler`, which we don't care about.

Reply:
Thanks, that makes sense.